PWGGA/GammaConv/AliAnalysisTaskEtaToPiPlPiMiGamma.cxx

```diff
@@ -657,7 +657,7 @@ void AliAnalysisTaskEtaToPiPlPiMiGamma::UserExec(Option_t *){
 	if(!fV0Reader){printf("Error: No V0 Reader");return;} // GetV0Reader
 
 	Int_t eventQuality = ((AliConvEventCuts*)fV0Reader->GetEventCuts())->GetEventQuality();
-	if(InputEvent()->IsIncompleteDAQ()==kTRUE) eventQuality = 2;  // incomplete event
+	if(InputEvent()->IsIncompleteDAQ()==kTRUE || fV0Reader->GetErrorAODRelabeling()) eventQuality = 2;  // incomplete event
```

Comment:
As we discussed the eventQuality should be 9 if AODRelabeling failed, please fix this for all tasks

Reply:
I am confused, when we discussed about eventQuality=9 we were talking about the time range cut, which is necessary because of problems in the TPC. Are you suggesting to use it for the AOD relabeling issue as well?

Reply:
You are right, I mixed up the two topics - "2" is totally fine here